### PR TITLE
init /auth page

### PR DESCRIPTION
**Pull request by wangkirin**
init /auth page

### CODE REVIEW

File: handler/user.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package handler
+
+import (
+	"github.com/Unknwon/macaron"
+)
+
+func InitAuthHandler(ctx *macaron.Context) {
```
Comment (genedna) [inline handler/user.go]: Why named `InitAuthHandler`, I think should be `AuthHandler` .

Comment (wangkirin) [inline handler/user.go]: I will correct it in next submit :)

File: external/views/auth/signin.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        </div>
+    </div>
+    <div class="form-group form-actions">
+        <div class="col-xs-12 text-right">
+            <button class="btn btn-sm btn-primary"><i class="fa fa-sign-in fa-fw" ng-show="!submitting"></i><i class="fa fa-refresh fa-spin" ng-show="submitting"></i> Login to Dashboard</button>
+        </div>
+    </div>
+    <div class="form-group">
+        <div class="col-xs-12 text-center">
+            <a href=""><small>Create a new account</small></a>
```
Comment (genedna) [inline external/views/auth/signin.html]: Should be add a `href` to change the `signup` view.

File: views/auth.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1.0">
+
+    
+    <link rel="stylesheet" href="../css/bootstrap.css" media="screen">
+  
+  </head>
+  <body ng-app="auth">
+    
+    <!-- Auth Container  -->
+    <div id="login-container" class="animation-fadeIn">
```
Comment (genedna) [inline views/auth.html]: Now the `div` is a full page form, change to the form in the center of page. You could add simple css for the requirement.

File: views/auth.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  <head>
+    <meta charset="utf-8">
+
+    <title>ContainerOps Platform - Auth</title>
+    <meta http-equiv="content-type" content="text/html;charset=UTF-8">
+   
+    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1.0">
+
+    
+    <link rel="stylesheet" href="../css/bootstrap.css" media="screen">
```
Comment (genedna) [inline views/auth.html]: Why add  bootstrap.css ?
You should be add `css` file from `bower_components`.

**Pull request merged**